File: csharp/7-kyu/complementary-dna/solution.cs
public class DnaStrand 
    {
        public static string MakeComplement(string dna)
        {
            var result = "";
            for (var i = 0; i < dna.Length; i++)
            {
              if (dna[i] == 'G')
                result += 'C';
              else if (dna[i] == 'C')
                result += 'G';
              else if (dna[i] == 'T')
                result += 'A';
              else if (dna[i] == 'A')
                result += 'T';
            }
            
            return result;
        }
    }

File: javascript/7-kyu/complementary-dna/solution.js
function DNAStrand(dna) {
  let dnaArray = dna.split("");

  dnaArray.forEach((letter, index) => {
    switch (letter) {
      case "A":
        dnaArray[index] = "T";
        break;
      case "T":
        dnaArray[index] = "A";
        break;
      case "G":
        dnaArray[index] = "C";
        break;
      case "C":
        dnaArray[index] = "G";
        break;
    }
  });

  return dnaArray.join("");
}
